example/pubspec.yaml

```diff
@@ -18,6 +18,8 @@ dev_dependencies:
 
 flutter_icons:
   image_path: "assets/images/icon-710x599.png"
+  image_path_android: "assets/images/icon-710x599-android.png"
+  image_path_ios: "assets/images/icon-710x599-ios.png"
```

Comment:
Do both icon-710x599-android.png and icon-710x599-ios.png exist? If not, ensure they both exist. Example should allow anyone to test out the package quickly (myself included).